sdk/tests/conformance/context/methods.html

```diff
@@ -161,47 +161,51 @@
 
 // Properties to be ignored because they were added in versions of the
 // spec that are backward-compatible with this version
-var ignoredMethods = [
+const ignoredMethods = [
   // There is no official spec for the commit API yet, the proposal link is:
   // https://wiki.whatwg.org/wiki/OffscreenCanvas
   "commit"
 ];
 
+const optionalMethods = [
+  "makeXRCompatible"
```

Comment:
Should the presence of this method be tied to the existence of some other method or global interfaces?

Reply:
I thought about that, but I decided it was the place of the WebXR tests to check for conformance there. I consider this patch to simply be a workaround for a well-known method name.